src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -145,6 +157,16 @@ public String getCurrentUserSessionToken() {
 	}
 	
 	@Override
+	public UserBundle getCurrentUserBundle() {

```

Comment:
This method will be the foundation of upcoming work on ACTs: 

The user bundle can tell us if the current user is part of the ACT. 
One step that I think we need to make this work is that on Synapse log in, we should kick off a process to ask for the user bundle of the current user and store it in a global place (authenticationController?). When asking for the current user bundle, it should verify that it is set and matches the current user id from the session data.
**Once that's done, the main feature can be implemented.** I'll draw my thoughts on a whiteboard and attach them shortly, but I think a new command would be available from the Tools menu (that would only be shown if the current user is in the ACT). When clicked, a dialog would pop up where the ACT member would select the Access Requirement and User.